### PR TITLE
Lang artificial

### PR DESCRIPTION
**Pull request by xhero**

### CODE REVIEW

File: locales/de.yml
Comment (ahankinson) [inline locales/de.yml:49]: "and"? Should this be "und"?

File: locales/pt.yml
Comment (ahankinson) [inline locales/pt.yml:49]: Same here... keep "and"?

Comment (jenniferward) [reply locales/pt.yml:49]: I don't have the official PT text.

File: locales/de.yml
Comment (xhero) [reply locales/de.yml:49]: @MFalletta This is your call here

File: locales/pt.yml
Comment (xhero) [reply locales/pt.yml:49]: I did not notice the languages were in English still, I can go back to "other" but then we have some that say "outro" and others that say "other"

Comment (jenniferward) [reply locales/pt.yml:49]: Maybe best have your formulation with English and "Translation missing" so we can find it later and ask.

Comment (ahankinson) [reply locales/pt.yml:49]: It was more just to point out that this still existed. I am happy to merge if this is OK, but thought I would point it out in case it had simply been overlooked.

File: locales/de.yml
Comment (MFalletta) [reply locales/de.yml:49]: DE: Kreolisch und Kauderwelsch


horrible languages

Comment (xhero) [reply locales/de.yml]: Fixed!

File: locales/pt.yml
Comment (xhero) [reply locales/pt.yml]: And this one too

**Review by ahankinson — APPROVED** [review]

**Pull request merged**
